packages/orchestrator-ui-components/src/types/types.ts

```diff
@@ -165,6 +165,38 @@ export type Process = {
     lastModified: string;
 };
 
+export enum StepStatus {
+    SUCCESS = 'success',
+    FAILED = 'failed',
+    PENDING = 'pending',
+    RUNNING = 'running',
+}
+
+export interface ProcessDetail {
+    processId: Process['processId'];
+    status: Process['status'];
+    createdBy: Process['createdBy'];
+    started: Process['started'];
```

Comment:
You could replace this whole block with the Pick utility type

Reply:
This does the same and gives you a better overview of the type IMO